kibble/api/prices.go

```diff
@@ -126,11 +126,31 @@ func processPrices(details prices, site *models.Site, itemIndex models.ItemIndex
 		}
 	}
 
+	// loop over the retrieved list of films that belong to plans
+	// assign a slice of maps to each film e.g. [{"plan/123" : prices }, ...]
+	for _, filmPlans := range details.Plans {
+		if film, ok := site.Films.FindFilmBySlug(filmPlans.Item); ok {
```

Comment:
This seems like it could be a fairly inefficient way to get a single film from the list of all films which aren't indexed.

Reply:
Agreed. I don't think this can be avoided without restructuring kibble though.

Issue is the pattern that kibble uses e.g.
` [{ Item: "/film/1" ... },{ Item: "/film/2" ... }]`

Causing the need to query every item rather than just square bracket notation  e.g. 
`{"/film/1": {...},{ "/film/2": {...}}`

There's probably an original intention or good reason for this but unsure.

note `/pricing/v2/prices/show_multiple` also uses this pattern.